lib/View/bookmarks_screen.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/View/utils/custom_bottom_nav_bar.dart';
+import 'package:gocast_mobile/View/utils/video_card_view.dart';
+class BookmarksScreen extends ConsumerWidget {
```

Comment:
Rename to `PinnedCourses` or something similar to avoid confusion with the actual bookmarks (=timestamps)